databricks/koalas/groupby.py

```diff
@@ -90,7 +91,7 @@ def _agg_columns_scols(self):
         return [s.spark.column for s in self._agg_columns]
 
     @abstractmethod
-    def _apply_series_op(self, op, should_resolve: bool = False):
+    def _apply_series_op(self, op, should_resolve: bool = False, is_cum_func: bool = False):
```

Comment:
nit: I'm feeling a little weird about the naming `is_cum_func`. We could name it `numeric_only` for more generality. I don't think we should fix it at once, but let's rename it when we have another chance.

Reply:
Ah, I think I can revise right away before we forget 😂.
Will quickly fix it. Thanks for the suggestion!